Header header
std_msgs/Float64MultiArray mean
std_msgs/Float64MultiArray std_dev
std_msgs/Int64MultiArray counter
std_msgs/Int64MultiArray land

float64 grid_size
float64 cell_size

geometry_msgs/Vector3 curr_pos_index
